faunadb-java-dsl/src/main/java/com/faunadb/client/types/Value.java

```diff
@@ -348,7 +348,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-      return format("ObjectV(%s)", values);
+        return values.entrySet()
+            .stream()
+            .map(e -> String.join(" = ", Arrays.asList(e.getKey(), String.valueOf(e.getValue()))))
```

Comment:
Why the `=` symbol? Will it be easier to understand if we mimic JSON objects?

Reply:
'=' is only because Java uses = in `Map.toString()` - I don't have an opinion about what the delimiter should be. It can be any thing you like. :)

Reply:
In general, I think people understand our types as being JSON types. We even had people serializing our values to JSON in order to print them out as debuting strings before.

Reply:
So, change to ':' then?

Reply:
I personally prefer `:`.